schemas/v1/config/interventions.json

```diff
@@ -0,0 +1,104 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/imperialCHEPI/healthgps/main/schemas/v1/config/interventions.json",
+    "type": "object",
+    "properties": {
+        "active_type_id": {
+            "anyOf": [
+                {
+                    "type": "string"
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        "types": {
+            "type": "object",
+            "additionalProperties": {
+                "type": "object",
+                "properties": {
+                    "active_period": {
+                        "type": "object",
+                        "properties": {
+                            "start_time": {
+                                "type": "integer"
+                            },
+                            "finish_time": {
+                                "anyOf": [
+                                    {
+                                        "type": "integer"
+                                    },
+                                    {
+                                        "type": "null"
+                                    }
+                                ]
+                            }
+                        },
+                        "required": [
+                            "start_time",
+                            "finish_time"
+                        ],
+                        "additionalProperties": false
+                    },
+                    "impacts": {
+                        "type": "array"
+                    },
+                    "impact_type": {
+                        "type": "string"
+                    },
+                    "dynamics": {
+                        "type": "array",
+                        "items": {
+                            "type": "number"
+                        }
+                    },
+                    "coefficients": {
+                        "type": "array",
+                        "items": {
+                            "type": "number"
+                        }
+                    },
+                    "coverage_rates": {
+                        "type": "array",
+                        "items": {
+                            "type": "number"
+                        }
+                    },
+                    "coverage_cutoff_time": {
+                        "type": "integer",
+                        "minValue": 0
```

Comment:
I don't really know, but it seems the keyword for setting a minimum value is `minimum`, rather than `minValue`. See here:
https://json-schema.org/understanding-json-schema/reference/numeric#range

Reply:
Good spot! I'm not sure how I managed that :facepalm:. We also have a `pre-commit` hook to check the schemas are valid, but it seems you can add extra properties and it won't complain :shrug: